<launch>
    <arg name="imu_name" default="imu" />
    <arg name="positions_file" value="$(find riptide_controllers)/cfg/vehicle_properties_gazebo.yaml" />

    <node pkg="riptide_hardware" type="depth_processor" name="depth_processor" >
        <param name="post_IIR_LPF_bandwidth" type="double" value="$(arg post_IIR_LPF_bandwidth)" />
        <param name="sensor_rate" type="double" value="22" />
  </node>

    <node pkg="riptide_hardware" name="imu_processor" type="imu_processor" output="screen" >
        <rosparam command="load" file="$(arg settings)" />
        <param name="imu_name" type="string" value="$(arg imu_name)" />
        <param name="post_IIR_LPF_bandwidth" type="double" value="17" />
    </node>

  <node pkg="riptide_hardware" type="dvl_processor" name="dvl_processor" output="screen">
    <param name="properties_file" type="string" value="$(arg positions_file)" />
</launch>